core/services/workflows/monitoring.go

```diff
@@ -13,10 +13,18 @@ import (
 )
 
 var registerTriggerFailureCounter metric.Int64Counter
+var triggerWorkflowStarterErrorCounter metric.Int64Counter
 var workflowsRunningGauge metric.Int64Gauge
 var capabilityInvocationCounter metric.Int64Counter
+var capabilityFailureCounter metric.Int64Counter
+var workflowRegisteredCounter metric.Int64Counter
+var workflowUnregisteredCounter metric.Int64Counter
+var workflowExecutionFinishedCounter metric.Int64Counter
 var workflowExecutionLatencyGauge metric.Int64Gauge // ms
 var workflowStepErrorCounter metric.Int64Counter
```

Comment:
is `capabilityFailureCounter` a subclass of `workflowStepErrorCounter`?

Reply:
The idea here is that we know “how many times this workflow failed because of capability X”... 🤔 @patrickhuie19

Reply:
That is likely better tracked through logging (for now, custom messages). Metrics show us high level symptoms, and then we can use logs and traces to drill in

Reply:
cc: @cedric-cordenier

Reply:
I agree with that long-term but this is the list of metrics agreed with dev-services